documentation/docs/developers/strategies/twap.md

```diff
@@ -0,0 +1,53 @@
+# TWAP
+
+Unlike the previous tutorial strategies which demonstrated straight-forward tasks like adding configurations and using clock ticks to periodically perform action, this strategy will introduce you to implement a more complicated algorithm. The tools demonstrated in this strategy is : 
+1) Maintaining some states over clock ticks to retain important information that will decide algorithm's next action 
+2) Quantizing order size depending of user's specification of how the user wants order to be executed.
+
+## Overview
+
+TWAP strategy provides the guide for you to add these features : 
+
+* Divide up an order into *n* individual orders
+* Add time delay between each individual orders
+
+Note that unlike Simple Trade strategy which places one order of entire quantity, the number of order is dynamically determined by user's input. Since one order can be placed over a single clock tick, there needs to some sort of information about order 'state' that will indicate when the order should stop.
+
+## Config
+
+To implement a more sophisticated algorithm, you probably want to ask the user for extra information that you need to feed into the algorithm. This can be done by adding more fields to the `config_map` file :
```

Comment:
We can keep the wording simple here. e.g. "Here are the user configurable parameters for the TWAP strategy"